docs/tutorials/tutorial-custom-domain-with-istio.md

```diff
@@ -347,7 +363,7 @@ spec:
         ports:
         - containerPort: 80
 ---
-apiVersion: networking.istio.io/v1
+apiVersion: networking.istio.io/v1beta1
```

Comment:
Why are you not staying with the current version here?

Reply:
I do not know why the version got changed, maybe it's an issue of documentation versioning. I will fix it.